frontend/src/components/WarningModal.tsx

```
@@ -0,0 +1,35 @@
+        return null;
+    }
+
+    return (
+        <div className={styles.modalOverlay}>
+            <div className={styles.modalContent}>
+                <p>
+                    Your solution has not passed all code checks. Are you sure
+                    you want to skip to the solution?
+                </p>
+                <div className={styles.buttonContainer}>
+                    <button onClick={onContinue}>Yes</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)